src/ApiService/ApiService/Functions/Config.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+
+namespace Microsoft.OneFuzz.Service.Functions;
+
+public class Config {
+    private readonly ILogTracer _log;
+    private readonly IOnefuzzContext _context;
+
+    public Config(ILogTracer log, IOnefuzzContext context) {
+        _log = log;
+        _context = context;
+    }
+
+    [Function("Config")]
+    public Async.Task<HttpResponseData> Run(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "GET", Route = "config")] HttpRequestData req) {
```

Comment:
>Route = "config" [](http://example.com/codeflow?start=58&length=16)

I don't think this is strictly needed as it's the same as the name of the function.

Reply:
Removed.